using-mobile-extensions/mobile-core/lifecycle/README.md

```diff
@@ -76,14 +76,46 @@ The Lifecycle framework provides valuable information about your user's current
 
 {% tab title="iOS" %}
 1. Register the Lifecycle extension: In your app's `didFinishLaunchingWithOptions` function register the Lifecycle extensions
+2. Add the `lifecycleStart` and `lifecyclePause` methods to the delegate methods following the sample code:
```

Comment:
please change to read as follows:

"...methods to the delegate methods." --> 1st line
"Here is the code sample:" --> next line

Thanks.